modules/nlp/lexers/omnilang-lexer/src/main/java/org/daisy/pipeline/nlp/lexing/omni/impl/OmnilangLexer.java

```diff
@@ -71,6 +71,15 @@ public void addLang(Locale lang) throws LexerInitException {
 			//replace "!)" with ",)" to prevent it from detecting a new sentence
 			input = input.replaceAll("[؟:?‥!…។៕。]\\)", ",)");
 
+			if (input.matches(".*https?://.*")) {
+				int offset = 0;
+				while((offset = input.indexOf('?', offset)) != -1) {
+					if (input.charAt(offset) == '?' && input.charAt(offset + 1) != ' ') {
```

Comment:
Couldn't this cause a StringIndexOutOfBoundsException?

Reply:
Hi @bertfrees 

Great catch, you are totally correct. Added a test and also a quick fix to the code.

Best regards
Daniel